fw-child/resources/js/child-functions.js

```diff
@@ -686,5 +686,18 @@ const $ = jQuery;
     //
 
     $('body').removeClass('spinner-on');
+
+    //
+    // Add 'scrolled' class on body after 50px scroll.
+    //
+
+    $( window ).on( 'scroll', function() {
+      if ( $( this ).scrollTop() > 50 ) {
+        $( 'body' ).addClass( 'scrolled' );
+      } else {
+        $( 'body' ).removeClass( 'scrolled' );
+      }
+    });
```

Comment:
The `scrolled` class is not added when loading the page, so if we scroll the page, then reload it, the page is shown already scrolled, but the class is not added.

It creates the problem in this image:
![image](https://github.com/user-attachments/assets/d0b8b48f-fc9d-4fce-8787-c72c305e8ef1)